res/css/views/dialogs/_ForwardDialog.scss

```diff
@@ -54,6 +54,10 @@ limitations under the License.
             display: none;
         }
 
+        .mx_EventTile_e2eIcon_warning {
```

Comment:
While we're here, I think we can merge all the `display: none` into one selector, e.g.:

```css
.mx_EventTile_e2eIcon_unencrypted,
.mx_EventTile_e2eIcon_warning,
...
.mx_MFileBody_download {
	display: none
}
```